ote/src/cljs/ote/app/routes.cljs

```diff
@@ -143,7 +143,12 @@
                      event-to (if (vector? event-to) event-to [event-to])
                      orig-app app
                      app (merge app navigation-data)
-                     win-location (subs (str "/" (.. js/window -location -hash)) 1)]
+                     win-location (subs (.. js/window -location -hash) 1)
+                     win-location (if (boolean
```

Comment:
This could be `if-let [page (#{:register :reset-password} (:page navigation-data)))`